src/inputs/parseViteConfigReporters.ts

```diff
@@ -0,0 +1,45 @@
+import path from 'node:path';
+import * as core from '@actions/core';
+import { promises as fs } from 'fs';
+import { VitestConfig } from "../types/VitestConfig";
+import { Reporter } from "vitest";
+
+export const parseViteConfigReporters = async (vitestConfigPath: string): Promise<Reporter | Reporter[]> => {
+	try {
+		const resolvedViteConfigPath = path.resolve(process.cwd(), vitestConfigPath);
+		const rawContent = await fs.readFile(resolvedViteConfigPath, 'utf8');
+		const vitestConfig: VitestConfig = JSON.parse(rawContent);
```

Comment:
I am pretty certain that this won't work, as the vitestConfig is usually not a `.json` File, but mostly is a `.ts` or `.js` file and can have various formats. So we either need to take the simple approach and handle this file as a `string`, looking for the `reporters` configuration through string extraction (or regexp) - or see if vite has some API to handle the parsing of this configuration.

For the latter, I found https://vitejs.dev/guide/api-javascript.html#loadconfigfromfile - but I am not sure if this is usable or will reliably work, given that it might even have to compile a `.ts` file... but it could be worth a shot.